Guacamole/Assets/Scripts/MoleMovement.cs

```diff
@@ -0,0 +1,38 @@
+﻿ using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoleMovement : MonoBehaviour
+{
+    public Transform pos1, pos2;
+    public float speed;
+    public Transform startPos;
+
+    Vector3 nextPos;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        nextPos = startPos.position; 
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if(transform.position == pos1.position)
+        {
+            nextPos = pos2.position; 
+        }
+        if(transform.position == pos2.position)
+        {
+            nextPos = pos1.position; 
+        }
+
+        transform.position = Vector3.MoveTowards(transform.position, nextPos, speed * Time.deltaTime);
+    }
+
+    private void OnDrawGizmos()
+    {
+        Gizmos.DrawLine(pos1.position, pos2.position);
+    }
```

Comment:
This Gizmos thing is super interesting. I wonder if we could use it for other things as well.